scripts/docs_version_pragma_check.sh

```diff
@@ -124,7 +124,7 @@ function findMinimalVersion()
         fi
     done
 
-    if [[ "$version" == "" ]]
+    if [[ "$version" == "" && "$greater" == false ]]
```

Comment:
This does not seem like a correct change. I'm not sure what happens if `version` is empty but the script was not meant to continue in that case. This basically allows anything after `>` in the pragma and ignores it unless it happens to match one of the released versions.

Reply:
BTW, we should really rewrite this script in something sane like Python. Trying to read this with all kinds of variables being used as flags and implicit globals like `version` just hurts :( Every time I have get back to this I have to stare at the code for a good while before I understand what it does, even though it's not even that long.